Unix/server/server.c

```diff
@@ -894,6 +895,13 @@ static void GetConfigFileOptions()
             if (SetPathFromNickname(key, value) != 0)
                 err(ZT("SetPathFromNickname() failed"));
         }
+        else if (strcasecmp(key, "NtlmCredsFile") == 0)
+        {
+            if (value)
+            {
+                s_opts.ntlmCredFile = PAL_Strdup(value);
```

Comment:
So, as I said above, what happens if there are spaces in the filename? Or what if there is a trailing space here on the line? I don't see code to get rid of leading/trailing spaces (or quote characters). If I'm correct, then you either need to improve handling here for parsing or be quite explicit in your documentation.

It would be easy for a trailing space to mess things up, and not be clear why NTLM is not working properly.